src/components/screens/Service.tsx

```diff
@@ -51,6 +52,8 @@ export const Service = ({
   const { globalSettings } = useContext(SettingsContext);
   const { settings = {} } = globalSettings;
   const { personalizedTiles = false, tileSizeFactor = 1 } = settings;
+  const { appDesignSystem } = useContext(ConfigurationsContext);
+  const { serviceTiles = {} } = appDesignSystem;
```

Comment:
why does this happen here, just to pass it to `ServiceTile`? shouldn't it be placed directly in `ServiceTile` to use the styles there?

Reply:
I find it better to place it here for optimization. If I place it in the ServiceTile component and we have for example 10 service tiles, a request is sent to the context api for all of them. But here only one request is sent and it is applied for all of them.

Reply:
ok, but then please find a better prop name for `serviceTileStyles` as it is not a style object as we use it for styling elements but a self defined object of configs

```
const { fontStyle = {}, iconStyle = {}, numberOfLines, tileStyle = {} } = serviceTileStyles;
```

so as it is named `const { serviceTiles = {} } = appDesignSystem;` from the global configs, why not using that name further as the prop?

Reply:
I honestly thought `serviceTileStyles` would be more descriptive. But of course it can also be used as `serviceTiles`.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1307/commits/c34455a461b7746ffd807194727c8dd2162f62b0